llarp/config/config.cpp

```diff
@@ -266,13 +266,21 @@ namespace llarp
   void
   MetricsConfig::fromSection(string_view key, string_view val)
   {
-    if(key == "disable-metrics")
+    if(key == "enable-metrics")
     {
-      disableMetrics = true;
+      disableMetrics = IsFalseValue(val);
```

Comment:
I think if we’re doing it like this, we should rename the variable to `enableMetrics`

Reply:
i added an option for that too

Reply:
I mean, if metrics are disabled by default, we should make the variable “enableMetrics”, with a default value of false. 

Positive bools are easier to understand imo.

Reply:
i agree